package.json

```diff
@@ -38,7 +38,7 @@
     "testee": "^0.3.0-pre.2"
   },
   "dependencies": {
-    "qunitjs": "^1.17.1",
+    "qunitjs": "^2.3.2",
```

Comment:
Is it possible to do a range in order to support both?  `>=1.17.1 || ^2.3.2` should work I think.